README.md

```diff
@@ -1,3 +1,75 @@
-This repo contains the code for some Lucid built Lucid Editor Extensions and Lucid REST API integrations.
-Check out each subfolder to see individual extensions and integrations and how they work.
-Here is our [official documenation](https://developer.lucid.co/).
+# Sample Lucid Extensions
+
+This repo contains example applications built using [Lucid Software's APIs](https://developer.lucid.co/).
+
+## AI Prompt Flow
+
+An interactive prompt building aid for working with AI built on the Extension API.
+Uses custom shapes, network requests, and an Angular frontend to create an interactive experience.
+
+![AI Prompt Flow](https://lucid.co/static/cd8ee8e60b1e6b4f92938d77e9e6118f/content.png)
+
+You can find more information about the application, and install it to use for yourself at its [marketplace listing](https://lucid.co/marketplace/1d2a2200/ai-prompt-flow).
+
+## Asana
+
+Our Lucid cards for Asana integration is a production ready implementation of a cards integration built to import and manage Asana tasks in Lucidspark.
+The integration uses Lucid's Extension API, and leverages the built in cards framework to allow users to bring Asana data into Lucidspark and to keep that data up to date when changes are made.
+
+![Lucid Cards for Asana](https://lucid.co/static/b20c5a40f333fba7b7799519d8fbd2e0/content.png)
+
+You can find more information about the integration, and install it to use for yourself at its [marketplace listing](https://lucid.co/marketplace/944af3b8/lucid-cards-for-asana).
+
+## BPMN Converter
+
+A tool for importning BPMN formatted files into Lucid using the REST API.
+Uses our JSON import enpoints to create documents programatically.
+
+## Example Card Integration
+
+An example card integration which uses mocked data.
+This is a great place to start if you're building a cards integration!
+
+![Example Card Integration](https://cdn-cashy-static-assets.lucidchart.com/open-source-github-repositories/sample-lucid-extensions/example_cards_entrypoint.png)
+
+## Example REST API Integration
+
+A NextJS fullstack application which connects to Lucid user data using OAuth2 and Lucid's REST API.
+
+![Example REST API application demo](https://cdn-cashy-static-assets.lucidchart.com/open-source-github-repositories/sample-lucid-extensions/example_rest_api_demo.gif)
+
+## Example Shape Library
+
+A collection of custom shapes that can be used in Lucidchart.
+These shapes cover data linking, conditional formatting, images, dynamic rendering and more.
+You can find more information about designing your own custom shapes [here](https://developer.lucid.co/custom-shapes/).
+
+![Example shape library](https://cdn-cashy-static-assets.lucidchart.com/open-source-github-repositories/sample-lucid-extensions/example_shape_library.png)
+
+## Google Embed
+
+An integration which brings Google Docs documents into Lucidchart and Lucidspark as expandable links using our Extension API.
```

Comment:
```suggestion
An integration which brings Google documents into Lucidchart and Lucidspark as expandable links using our Extension API.
```

Reply:
I can see this one either way. Google Docs is technically the name for the product we're integrating with so I'm hesitant to remove that identifier. I could see removing "documents" though... Unless others feel strongly I think I'll leave this as is for now.